src/analyses/commonPriv.ml

```diff
@@ -160,33 +190,95 @@ struct
   end
 end
 
-module ConfCheck =
+module type PerMutexTidGArg = sig
+  val exclude_not_started: unit -> bool
+  val exclude_must_joined: unit -> bool
+end
+
+module PerMutexTidG (Conf:PerMutexTidGArg) (LD:Lattice.S) =
```

Comment:
This doesn't just provide `G`, but `D`, etc as well, so maybe `PerMutexTidBase` is more appropriate.

Reply:
Renamed to `PerMutexTidCommon`